### ISSUE TITLE
json request per second

### ISSUE THREAD

**Issue by bits16** [vozlt/nginx-module-vts#102]
Hello
How to draw a separate conclusion of request per second for a certain domain? through status/control? cmd =

**Comment by vozlt** [comment]
Yes, See the [https://github.com/vozlt/nginx-module-vts#to-get-each-zones](https://github.com/vozlt/nginx-module-vts#to-get-each-zones)
For example, to acquire a specific domain zone within a server zone:
Syntax: `http://localhost:1981/status/control?cmd=status&group=server&zone={domain}`
Example: `$ curl http://localhost:1981/status/control?cmd=status&group=server&zone=example.org`

Thanks.
